pages/dkp/konvoy/2.1/choose-infrastructure/aws/advanced/prerequisites/index.md

```diff
@@ -17,7 +17,14 @@ Before you begin using Konvoy, you must have:
 - [kubectl][install_kubectl] for interacting with the running cluster.
 - A valid AWS account with [credentials configured][aws_credentials].
 
-<p class="message--note"><strong>NOTE: </strong>On macOS, Docker runs in a virtual machine. Configure this virtual machine with at least 8GB of memory.</strong></p>
+<p class="message--note"><strong>NOTE: </strong>On macOS, Docker runs in a virtual machine. Configure this virtual machine with at least 8GB of memory.</p>
+
+If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on a [CentOS 7 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
+
+<p class="message--note"><strong>NOTE: </strong>
+Using these default images work, but due to missing optimizations, the created cluster will have certain limits.
+We suggest using <a href="../../../../image-builder/create-ami/">Konvoy Image Builder to create a custom AMI</a> to take advantage of enhanced cluster operations.
```

Comment:
As an old punster used to say: "Be Pacific, not Atlantic".     :-)   I.e. the words here are very vague and not very helpful - what are the 'certain limits'?   If we're going to mention 'missing optimizations' and 'limits', we really need to specify them here, or at least give the reader a clue as to what that might mean.

Instead, I suggest we just point out that like many 'default' configs, they are usually not well suited for production use cases:

```suggestion
The AMI images used in the default configuration work, but they are general purpose, and not recommended for production workloads.  We suggest using the <a href="../../../../image-builder/create-ami/">Konvoy Image Builder to create a custom AMI</a> for production workloads.
```

Reply:
I'm with @cneth and support his suggestion. 

Although to me its not a strong enough push but thats a different story by having a default. So this sounds like a really good compromise with good wording